### ISSUE TITLE
MAS 4.1.2: Elements must only use allowed ARIA attributes”( ARIA attribute is not allowed: aria-orientation="horizontal")

### ISSUE THREAD

**Issue by SaumyaGargMS** [OfficeDev/office-ui-fabric-react#11318]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

**Comment by khmakoto** [comment]
@SaumyaGargMS this issue is blank, so I'll close it. Please feel free to open another issue and fill out the corresponding information in there!

**Comment by dzearing** [comment]
We only have 2 instances of `aria-orientation` in our codebase, and I'm guessing this is referring to OverflowSet?

![image](https://user-images.githubusercontent.com/1110944/69762151-3be86b00-111e-11ea-9ebd-ab4e6c69354f.png)

@micahgodbolt What is the default role used? Do we have `OverflowSet` usage which doesn't pass in role? And if the wrong role is provided, is the attribute still set? (Because I think so, and I think that could be causing this violation.)

**Comment by SaumyaGargMS** [comment]
Hi,
PFA the actual detailed error. I am using a commandbar and commandbarbutton from fabric which is causing this issue.
![ariaOrientationError](https://user-images.githubusercontent.com/58243372/69776729-5d4f6280-11c3-11ea-964b-597b627c2130.PNG)

**Comment by khmakoto** [comment]
Hi @SaumyaGargMS could you provide us with a more concrete example of what you are doing with the `CommandBar` to get this error? I just ran the Accessibility Insights from Web tool on the `CommandBar` page in our website and it isn't throwing the error you're showing above, even though we're still using `aria-orientation` as you can see in the screenshot below:

![image](https://user-images.githubusercontent.com/7798177/69978219-f42c5f80-14e0-11ea-925f-a9020026deee.png)

**Comment by SaumyaGargMS** [comment]
Hi @Makoto Morimoto<mailto:notifications@github.com>,

I have used an OWA component that is internally using the CommandBar. Here is the code to the owa component -FilterableCommandBar<https://outlookweb.visualstudio.com/Outlook%20Web/_git/client-web?path=%2Fpackages%2Fcommon%2Fcontrols%2Fpackages%2Fcontrols%2Fowa-filterable-menu%2Flib%2Fcomponents%2FFilterableCommandBar.tsx&_a=contents&version=GBmaster>. The component is present in this package -
owa-filterable-menu, in the repo client-web. I have used the same component in Bookings here - Bookings FilterableCommandBar<https://outlookweb.visualstudio.com/Outlook%20Web/_git/client-web?path=%2Fpackages%2Fbookings%2Fpackages%2Fbookings-module%2Fcomponents%2Fbookings-calendar-surface%2Flib%2Fcomponents%2FCalendarSurfaceTimeBar.tsx&_a=contents&version=GBmaster>


The same component has also been used in OWA calendar. Also, PFA the complete accessibility insights report for the owa calendar.

[cid:image001.png@01D5A9BD.00F42110]


[A screenshot of a social media post  Description automatically generated]


As Bookings is currently open only to a small set of whitelisted users I am pointing you to Calendar Surface for easy access. Please let me know if you want access to Bookings portal as well.

Thanks,
Saumya

**Comment by khmakoto** [comment]
Hi @SaumyaGargMS, thank you for the information! I think I can safely say that this is not an issue with fabric but rather a misuse by whoever is using your `FilterableCommandBar` component that internally uses our `CommandBar`. I can see that the error Accessibility Insights throws at you says that the role is `presentation` in the `div` that has the problem. This makes sense given that `presentation` [is not a `role` that supports the `aria-orientation` property](https://www.w3.org/TR/wai-aria-1.1/#aria-orientation).

However, in the Fabric code we set the `role` of the `OverflowSet` used inside the `CommandBar` to whatever is provided by the user of the component, with our default role being `menubar`, which does support the `aria-orientation` property. Given this, I think the user of your component is setting `role=presentation`, which is in turn triggering this error in Accessibility Insights.

I hope I explained myself. Please let me know if I didn't and let me know if you need anything else. Thanks!
